x/oracle/types/params.go

```diff
@@ -24,7 +24,23 @@ func ParamKeyTable() paramtypes.KeyTable {
 
 // NewParams creates a new Params instance
 func NewParams() Params {
-	return Params{}
+	return Params{
+		// maximum number of transactions be submitted in one round from a validator
+		MaxNonce: 1,
```

Comment:
> Change MaxNonce to **MaxTransactionPerRound** for clearer expression of its meaning. 
> 
> Change Mode to **PriceConfirmationMode** for a more explicit indication of its purpose.

Reply:
We have this 'nonce' field set in message structure, and this MaxNonce is the limitation for that nonce, it also works like `nonce` in transaction but in this special case of oracle to distinguish transactions.
For mode it's not only confirmation(threshold or all messages received) but also how to update price for single validator with new price for deterministic/non-deteministic sources. We've go detailed comments for it, maybe don't put all details in the field name.